client/asset/dcr/dcr.go

```diff
@@ -759,6 +784,30 @@ func (dcr *ExchangeWallet) feeRate(confTarget uint64) (uint64, error) {
 	return 1 + uint64(atomsPerKB)/1000, nil // dcrPerKB * 1e8 / 1e3
 }
 
+// externalFeeEstimator gets the fee rate from the external API
+func externalFeeEstimator(nb uint64) (float64, error) {
+	url := externalApiUrl + "/utils/estimatefee?nbBlocks=" + strconv.Itoa(int(nb))
+	r, err := http.Get(url)
+	if err != nil {
+		return 0, err
+	}
+	body, err := ioutil.ReadAll(r.Body)
```

Comment:
`ioutil` is deprecated.

Also, we shouldn't use `ReadAll` without a limit on the read size (never trust a remote source).  Maybe consider a `io.LimitReader` with a fairly small limit, like a few kB, `1 << 14` for a high limit for this response.